src/processing/visualising.R

```diff
@@ -0,0 +1,143 @@
+# Organisation code selection ---------------------------------------------
+
+code_org <- "RNA"
+
+
+
+# Clinical Space ----------------------------------------------------------
+
+Trust_clinical_plot <- function(RNA){
+  
+  temp_clinical <- Clinical_Space %>%
+    filter(Trust_Code == code_org)
+  
+  ggplot(temp_clinical, aes(x = Site_Name)) +
+    geom_bar(aes(y = Non_CS, fill = "Non clinical"), stat = "identity") +
+    geom_bar(aes(y = CS_Other, fill = "Clinical"), stat = "identity") +
+    #scale_y_continuous(labels = scales::comma()) +
+    ylab("Gross internal space in metres squared") +
+    xlab("Site name") +
+    theme(axis.text.x = element_text(angle = 60, hjust = 1)) +
+    ggtitle("Gross Internal Space, split by clinical and non clinical") +
+    scale_fill_manual(values = c("blue", "lightblue"), name = "Dedicated floor space")
+  
+}
+
+Trust_clinical_plot("Test")
+
+
+
+# Age Profile -------------------------------------------------------------
+
+Trust_backlog_plot <- function(RNA){
+  
+  temp_backlog <- Age_Profile %>%
+    filter(Trust_Code == code_org)
+  
+  ggplot(temp_backlog, aes(x = Site_Name)) +
```

Comment:
An alternative to adding repeated `geom_bar` calls is to convert the dataset from wide format to long format using the `dplyr` function `gather`. You can then have a `ggplot` function that would look like:

`ggplot(temp_backlog_wide, aes(x = Site_Name, y = Proportion, fill = Age)) +`

Reply:
Actioned in a commit.